ci-operator/step-registry/openstack/conf/clouds/openstack-conf-clouds-commands.sh

```diff
@@ -40,7 +40,7 @@ new_application_credentials() {
 			"${*:---restricted}" \
 			--expiration "$(date -d "$APPLICATION_CREDENTIALS_EXPIRATION" +%Y-%m-%dT%H:%M:%S)" \
 			--description "PROW_CLUSTER_NAME=${CLUSTER_NAME} PROW_JOB_ID=${PROW_JOB_ID}" \
-			--format json --column id --column secret \
+			--format json --column ID --column Secret \
```

Comment:
Is this change needed? It feels out of place.

Reply:
I hope no, just testing and issue in openstack-conf-clouds step